index.Rmd

```diff
@@ -344,7 +344,7 @@ allyears=rbind(data2019,data2018,data2017,data2016,data2015, data2014, data2013,
 ```
 
 ```{r,message=F, warning=F,results="hide"}
-######################calculate the column percentages for county and tract levels
+######################calculate the column percentages for county and tract levels 
 allyears= transform(allyears,
```

Comment:
I'm wondering if you can set up some of these as loops instead of transforming each variable in turn.